configurations/dev.ts

```diff
@@ -42,28 +42,30 @@ const translations: Translations = {
 const preEngagementConfig: PreEngagementConfig = {
```

Comment:
This file's changes are only formatting. We should introduce **lint** to this project soon.

Reply:
I agree.  Can you add this as a new follow-up task after this project is done?